.rubocop.yml

```diff
@@ -83,6 +83,9 @@ Naming/FileName:
 Rails/EnumHash:
   Enabled: false
 
+Rails/SkipsModelValidations:
```

Comment:
@RamonaChristen This is only temporary because I think we'll have to find a way to comply with this rule. I invoke `touch` on the encryptable object in the `encryptables_spec` file. Rubocop doesn't like that at all, because invoking touch seems to skip the `encryptable` models validations.

Reply:
Just a little idea: You can also just skip the Rubocop validation for the single spec file if this will be the only place in which this will be used.